packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -470,3 +472,26 @@ function createFulltextProjection({
         resolveType,
     });
 }
+
+function addSharedFiltersToInterfaceImplementations(where: GraphQLWhereArg, implementations: Node[]) {
+    const interfaceSharedFilters = Object.entries(where || {}).filter(([key]) => key !== "_on");
+    implementations.forEach((node) => {
+        interfaceSharedFilters.forEach(([sharedFilterKey, sharedFilterValue]) => {
+            if (!Object.prototype.hasOwnProperty.call(where, "_on")) {
+                where["_on"] = {};
+            }
+            let nodeWhere = where["_on"][node.name];
+            if (nodeWhere) {
+                nodeWhere[sharedFilterKey] = nodeWhere[sharedFilterKey]
+                    ? nodeWhere[sharedFilterKey]
+                    : sharedFilterValue;
+            } else {
+                nodeWhere = {
+                    [sharedFilterKey]: sharedFilterValue,
+                };
+            }
+            where["_on"][node.name] = nodeWhere;
+        });
+    });
+    return where;
+}
```

Comment:
This is the kind of function that I would really start to see unit tests for - I really don't have 100% confidence in what this is doing just by reading it alone.